examples/rdkafka_performance.c

```diff
@@ -863,7 +863,7 @@ int main (int argc, char **argv) {
 	while ((opt =
 		getopt(argc, argv,
 		       "PCG:t:p:b:s:k:c:fi:MDd:m:S:x:"
-                       "R:a:z:o:X:B:eT:Y:qvIur:lA:OwNHH:")) != -1) {
+                       "R:a:z:o:X:B:eT:Y:qvIur:lA:OwNhH:")) != -1) {
```

Comment:
I'd like to (p)reserve -h for help.
Since the consumer-side -H header parsing is for performance measurements/comparisons only I think we can require an argument in the consumer mode too, e.g., `-H parse` and set if read_hdrs to 1 if optarg is "parse".